README.md

```diff
@@ -94,4 +97,104 @@ client.create_jobs(
     app_id="{project-id}",
     inputs=[{"image_url":"https://cdn.super.ai/cool-bulldog.jpg"},{"image_url":"https://cdn.super.ai/hot-dog-01.jpeg"}]
 )
-```
\ No newline at end of file
+```
+
+# Creating a data program
+
+## Requirements
+  - Have a superai `dataprogrammer` account. Please [contact us](mailto:dataprogrammer@super.ai) to create an account.
+  - Make sure you are running python >= `3.6` and < `3.9`. Python `3.9.1` is not yet supported.
+
+## Installation  
+  1. Create a virtualenv (we recommend to use [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) or conda)
+  2. Run `pip install --upgrade pip==20.1.1`
+  3. Run `pip install "awscli"`
+  4. Run `pip install --upgrade "superai>=0.1.0.a1"`
+  5. Run `superai env set -e sandbox` 
+  6. Run `superai login -u <user_email>`
+  7. Verify that the pip was configured correctly by running `pip config get global.index-url`
+     If the response is empty then run `superai login --show-pip -u <user_email>` and copy/paste the 
+     `pip config set...` command as indicated
+  8. Install superai in dataprogramming mode `pip install --upgrade "superai[dp]>=0.1.0.alpha1"`
```

Comment:
should this be `a1` at the end?

Reply:
Yes, that is the version being released

Reply:
Installation step 4. uses `a1` and step 8. uses `alpha1`. I think it should be consistent.